java/ql/lib/semmle/code/java/frameworks/Strings.qll

```diff
@@ -64,7 +64,14 @@ private class StringSummaryCsv extends SummaryModelCsv {
         "java.lang;StringBuilder;true;StringBuilder;;;Argument[0];Argument[-1];taint;manual",
         "java.lang;CharSequence;true;charAt;;;Argument[-1];ReturnValue;taint;manual",
         "java.lang;CharSequence;true;subSequence;;;Argument[-1];ReturnValue;taint;manual",
-        "java.lang;CharSequence;true;toString;;;Argument[-1];ReturnValue;taint;manual"
+        "java.lang;CharSequence;true;toString;;;Argument[-1];ReturnValue;taint;manual",
+        "java.util;StringJoiner;true;StringJoiner;;;Argument[0..2];Argument[-1];taint;manual",
+        "java.util;StringJoiner;true;add;;;Argument[0];Argument[-1];taint;manual",
+        "java.util;StringJoiner;true;add;;;Argument[-1];ReturnValue;value;manual",
+        "java.util;StringJoiner;true;merge;;;Argument[0];Argument[-1];taint;manual",
+        "java.util;StringJoiner;true;merge;;;Argument[-1];ReturnValue;value;manual",
+        "java.util;StringJoiner;true;setEmptyValue;;;Argument[-1];ReturnValue;value;manual",
+        "java.util;StringJoiner;true;toString;;;Argument[-1];ReturnValue;taint;manual"
```

Comment:
Might be useful to add a trailing comma to avoid a diff for this line when new lines are added:
```suggestion
        "java.util;StringJoiner;true;toString;;;Argument[-1];ReturnValue;taint;manual",
```